### PR TITLE
Hardening/823 attribute association

### PR DESCRIPTION
**Pull request by LeandroGuillen**
Fixing issue #823.

Now a registerContext request that uses associations must specify "attributeAssociations" instead of "attributes". This is to align it with discoverContextAvailability responses.

### CODE REVIEW

**Comment by kzangeli** [comment]
LGTM

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -7,3 +7,4 @@
        convenience operations "/v1/contextAttributes/{entity::id}" and are thus 100% identical.
 - Fix: The convenience operation "PUT /v1/contextAttributes/{entity::id}/attributes/{attribute::name}" had a bug
        about the metadata in the update, this error has been fixed.
+- Fix: A registerContext request that uses associations must specify "attributeAssociations" instead of "attributes" to be consistent with DiscoveContextAvailability messages (Issue #823).
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Could you change the wiki documentation to replace the old "attribute" occurrences by "attributeAssociations" in the right places, please?

Once it get done, tell so in this comment.

Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Confirmed by skype.

NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**